web_client/src/components/ControlPanel.vue

```diff
@@ -72,6 +72,7 @@ export default {
     if (!this.navigateToNextIfCurrentScanNull()) {
       this.switchLock(this.experimentId);
       window.addEventListener('keydown', (event) => {
+        if (['textarea', 'input'].includes(document.activeElement.type)) return;
```

Comment:
Does this also affect text inputs outside of the email modal dialog, such as free-form comments?

Reply:
Yes, this applies to all inputs and textareas in the DOM

Reply:
This is worthy of mention in release notes, so that user's don't think we broke hotkeys.